src/main/resources/reference.conf

```diff
@@ -460,11 +466,14 @@ resourceTypes = {
       read_policies = {
         description = "view all policies and policy details for the kubernetes application"
       }
+      set_parent = {
+        description = "set parent of kubernetes app"
+      }
     }
     ownerRoleName = "creator"
     roles = {
       creator = {
-        roleActions = ["delete", "connect", "update", "status", "stop", "start", "read_policies"]
+        roleActions = ["delete", "connect", "update", "status", "stop", "start", "read_policies", "set_parent"]
```

Comment:
in order to set a `google-project` as parent `creator` (the owner role) needs to have `set_parent`